invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar/external_resources.html

```diff
@@ -0,0 +1,48 @@
+{#
+Copyright (C) 2020 CERN.
+
+Invenio RDM Records is free software; you can redistribute it and/or modify
+it under the terms of the MIT License; see LICENSE file for more details.
+#}
+{%- if external_resources -%}
+  <section id="external-resource" aria-label="{{ _('External resources') }}" class="ui segment rdm-sidebar external resource">
+  <h2 class="ui small header">{{ _('External resources')}}</h2>
+  {# dynamically create the list of external resources #}
+  <div class="ui relaxed divided list">
+    {%- for section, external_resources in external_resources | groupby("content.section") -%}
+      <h5 class="ui small header">{{ section }}</h5>
```

Comment:
minor: I think we have to make `section` a required parameter/field, just to avoid all the issues with how/were to display the ones that have `section: None`.